app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -0,0 +1,83 @@
+module Katello
+  class Api::V2::HostSubscriptionsController < Katello::Api::V2::ApiController
+    before_filter :find_host
+    before_filter :check_subscriptions, :only => [:add_subscriptions, :remove_subscriptions]
+
+    api :GET, "/hosts/:host_id/subscriptions", N_("List a host's subscriptions")
+    param :host_id, Integer, :desc => N_("Id of the host"), :required => true
+    def index
```

Comment:
After some discussion, I added support for /katello/api/subscriptions without a quantity